packages/amplication-client/src/VersionControl/Commit.tsx

```diff
@@ -66,6 +66,7 @@ const Commit = ({ projectId, noChanges }: Props) => {
     commitUtils,
   } = useContext(AppContext);
 
+  const isProjectUnderLimitation = currentProject?.isUnderLimitation;
```

Comment:
Same, well you got the point :)
https://github.com/amplication/amplication/pull/7566/files#diff-46cc6f90a9c4c5614b6c68c13f5eebf1d57a5b3df9c08fe35621958da38659c0R74